### PR TITLE
Add --domain-parallelism flag to fetch

### PR DESCRIPTION
**Pull request by mdamien**
When using `minet fetch`, I was looking to use this option so I added to the CLI options

### CODE REVIEW

File: minet/cli/commands.py [hunk 1]
```diff
@@ -768,7 +768,13 @@ def check_dragnet():
                 'help': 'The http method to use. Will default to GET.',
                 'dest': 'method',
                 'default': 'GET'
-            }
+            },
+            {
```
Comment (Yomguithereal) [inline minet/cli/commands.py:772]: Flags are listed in alphabetical order.

**Comment by Yomguithereal** [comment]
Thanks @mdamien. Just a nitpicking about alphabetical order and I can merge :)

**Comment by mdamien** [comment]
hop, done :)

**Comment by Yomguithereal** [comment]
Will you be needing a fast pypi release? Or can we buffer this into next release?

**Comment by mdamien** [comment]
No worry for releasing on pypi, I'm using my local fork

**Pull request merged**